File: RolePlayGame/java/fixtures/my/examples/TeensFixture.java
package my.examples;

import fitlibrary.DoFixture;

public class TeensFixture extends DoFixture {
	
	public UserFixture insert(){
		return new UserFixture();
	}
	
	public TeenagersFixture teenagersAre(){
		return new TeenagersFixture();
	}
}

File: RolePlayGame/dotnet/fixtures/my.examples/TeensFixture.cs
using fitlibrary;

namespace my.examples {

	public class TeensFixture : DoFixture {
		
		public UserFixture Insert(){
			return new UserFixture();
		}
		
		public TeenagersFixture TeenagersAre(){
			return new TeenagersFixture();
		}
	}
}